study.md

```diff
@@ -54,5 +57,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+They are similar because they both iterate over things one by one but different because streams are potentially unlimited while enumerables iterate over the data and then stop.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.  An example of an enumerable that's infinite is one that returns a digit of π on each iteration.